sci-visualization/fityk/fityk-1.2.9.ebuild

```diff
@@ -0,0 +1,89 @@
+# Copyright 1999-2014 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: /var/cvsroot/gentoo-x86/sci-visualization/fityk/fityk-1.2.1.ebuild,v 1.3 2014/11/28 21:37:36 pacho Exp $
+
+EAPI=5
+
+WX_GTK_VER="3.0"
+PYTHON_COMPAT=( python{2_6,2_7,3_2,3_3,3_4} )
+
+inherit autotools-utils fdo-mime python-r1 wxwidgets
+
+DESCRIPTION="General-purpose nonlinear curve fitting and data analysis"
+HOMEPAGE="http://fityk.nieto.pl/"
+SRC_URI="https://github.com/wojdyr/${PN}/releases/download/v${PV}/${P}.tar.bz2"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+IUSE="gnuplot nlopt readline python static-libs wxwidgets"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+CDEPEND="
+	>=sci-libs/xylib-1
+	>=dev-lang/lua-5.1
+	nlopt? ( sci-libs/nlopt )
+	python? ( ${PYTHON_DEPS} )
+	readline? ( sys-libs/readline )
+	wxwidgets? ( x11-libs/wxGTK:${WX_GTK_VER} )"
+DEPEND="${CDEPEND}
+	dev-libs/boost
+	dev-lang/swig"
+RDEPEND="${CDEPEND}
+	gnuplot? ( sci-visualization/gnuplot )"
+
+src_configure() {
+	local myeconfargs=(
+		--docdir="${EPREFIX}/usr/share/doc/${PF}"
+		--disable-xyconvert
+		--disable-python
+		$(use_enable nlopt)
+		$(use_enable wxwidgets GUI)
+		$(use_with readline)
+	)
+	autotools-utils_src_configure
+	if use python; then
+		myeconfargs=(
+		--disable-xyconvert
+		--enable-python
+		--disable-nlopt
+		--disable-GUI
+		--without-readline )
+		python_foreach_impl autotools-utils_src_configure
+	fi
+}
+
+src_compile() {
+	autotools-utils_src_compile
+	python_copy_sources
```

Comment:
Unfortunately, I won't have time during the next to weeks to take care. I hoped it would be more straight forward ;)
Maybe, somebody other might take over. Should I just close this pull request or leave it, so that others can get the raw ebuild to work on it?